Mutagen.Bethesda.Analyzers.Skyrim/Extensions/CellExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Concurrent;
+using Mutagen.Bethesda.Plugins;
+using Mutagen.Bethesda.Plugins.Cache;
+using Mutagen.Bethesda.Skyrim;
+namespace Mutagen.Bethesda.Analyzers.Skyrim;
+
+public static class CellExtensions {
+    private static readonly ConcurrentDictionary<FormKey, IReadOnlySet<ILocationGetter>> CellLocationCache = new();
```

Comment:
Dont think this static dictionary will be safe to do.  Since its parent functions seem unused, let's remove for now?

Reply:
Yep, was and old optimization that I forgot to remove when copying over my code. The function itself will be used in future tests that I'll add so I'll keep that for now but the dictionary will go.